src/olympia/constants/permissions.py

```diff
@@ -0,0 +1,57 @@
+from collections import namedtuple
+
+
+AclPermission = namedtuple('AclPermission', 'app, action')
+
+# Special null rule.
+NONE = AclPermission('None', 'None')
+
+# Admin super powers.  Very few users will have this permission (2-3)
+ADMIN = AclPermission('Admin', '%')
+
+# Can view admin tools.
+ADMINTOOLS = AclPermission('AdminTools', 'View')

```

Comment:
Let's be consistent about this maybe and call the constant `ADMIN_TOOLS`? similar with the others below?